NetKAN/AirlockPlus-Localization.netkan

```diff
@@ -0,0 +1,20 @@
+{
+    "spec_version" : "v1.4",
+    "identifier"   : "AirlockPlus-Localization",
+    "name"         : "AirlockPlus Localization",
+    "abstract"     : "Localizations for AirlockPlus",
+    "author"       : "cake-pie",
+    "license"      : "restricted",
```

Comment:
Localizations are actually CC-BY-SA 4.0 (see [TRANSLATING.md](https://github.com/cake-pie/AirlockPlus/blob/42e57bbcbbcdb1c24bf6c6a50ad1d4b74e959d07/Localization/TRANSLATING.md))

Might get a bit weird because your bots could then automatically submit this to archive.org but won't do so for the actual mod itself.